### PR TITLE
small css changes

### PR DESCRIPTION
**Pull request by artemanufrij**
Signed-off-by: Artem Anufrij <artem.anufrij@live.de>

same margin and order:
screenshot_20170504_201927

### CODE REVIEW

**Comment by artemanufrij** [comment]
screenshot_20170504_202706

**Comment by artemanufrij** [comment]
#120

File: css/style.css [hunk 1]
```diff
@@ -241,35 +241,36 @@ button:hover {
...
 	border: 0;
 	background-color: transparent;
-	float: left;
 	min-height: initial;
 }
 
 .stack h2 button,
 .stack .stack-actions {
 	float: right;
+	margin: 3px 1px 3px 19px;
```
Comment (juliushaertl) [inline css/style.css]: 19px look a bit to stretched. That should be just the same as the margin on the top (inside the grey box).

Comment (artemanufrij) [inline css/style.css]: @juliushaertl it looks very poor...

screenshot_20170504_215812

Comment (artemanufrij) [inline css/style.css]: @juliushaertl we could move "description-icon" 1px to the right... for the symmetry

Comment (artemanufrij) [inline css/style.css]: My favourite:
screenshot_20170504_220649

**Comment by juliushaertl** [comment]
@artemanufrij Can you have a look? The delete button moves out of the container for long titles. Also the text should be vertically aligned to the center.

When fixed can you squash those commits into one and add the signoff stuff? ;)
See http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git#5201642 for the squashing.

**Comment by artemanufrij** [comment]
@juliushaertl do you mean like this:
screenshot_20170505_201322

File: css/style.css [hunk 2]
```diff
@@ -240,36 +240,38 @@ button:hover {
...
 .stack h2 button,
 .stack .stack-actions {
 	float: right;
+	margin: 0px 0px 0px 20px;
 }
 
 .stack h2 span {
-	float: left;
+	width: 100%;
+    white-space: normal;
```
Comment (pixelipo) [inline css/style.css]: something wrong with indentation here

Comment (artemanufrij) [inline css/style.css]: you are right...

**Comment by juliushaertl** [comment]
@artemanufrij I've squashed them for now.  

Btw. feel free to join #nextcloud-deck in IRC :wink:

**Comment by jancborchardt** [comment]
By the way, that Edit and Delete icon (presumably) should ideally be stacked vertically in the popover and have text next to it. See how we did it in the Mail app: https://github.com/nextcloud/mail/pull/344 :)

**Comment by juliushaertl** [comment]
@jancborchardt Makes sense.

@artemanufrij I think we already discussed that on irc. I'll open an issue for that.

**Pull request merged**